# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1273559518 -7200
#      Tue May 11 08:31:58 2010 +0200
# Node ID 483dbafc518cc24d8e473dc4e98e208c65099c13
# Parent  6fb954475e21432d95044476c8ae28906427d91e
fix bug in binary saving of lazy indices

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-05-11  Jaroslav Hajek  <highegg@gmail.com>
+
+	* ov-lazy-idx.cc (octave_lazy_index::save_binary): Fix arguments to
+	save_binary_data.
+
 2010-05-07  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/sqrtm.cc (do_sqrtm): Mark the result as triangular
 	where applicable.
 
 2010-05-10  Rik <octave@nomad.inbox5.com>
 
 	* DLD-FUNCTIONS/qr.cc: Split qrdelete test into high precision test 
diff --git a/src/ov-lazy-idx.cc b/src/ov-lazy-idx.cc
--- a/src/ov-lazy-idx.cc
+++ b/src/ov-lazy-idx.cc
@@ -171,17 +171,18 @@ bool octave_lazy_index::load_ascii (std:
     index = value.index_vector ();
 
   return ! error_state;
 }
 
 
 bool octave_lazy_index::save_binary (std::ostream& os, bool& save_as_floats)
 {
-  return save_binary_data (os, make_value (), value_save_tag, false, 0, save_as_floats);
+  return save_binary_data (os, make_value (), value_save_tag, 
+                           std::string (), false, save_as_floats);
 }
 
 bool octave_lazy_index::load_binary (std::istream& is, bool swap, 
                                      oct_mach_info::float_format fmt)
 {
   bool dummy;
   std::string doc;
 
